packages/material-ui/src/styles/createTypography.d.ts

```diff
@@ -16,23 +17,18 @@ export type TextStyle =
 export type Style = TextStyle | 'button';
 
 export interface FontStyle {
-  fontFamily: React.CSSProperties['fontFamily'];
+  fontFamily: CSSProperties['fontFamily'];
   fontSize: number;
```

Comment:
Is there a reason `FontStyle`'s `fontSize` shouldn't also be using `CSSProperties['fontSize']`?